slack-base/src/main/java/com/hubspot/slack/client/methods/params/conversations/ConversationsRepliesParamsIF.java

```diff
@@ -0,0 +1,17 @@
+package com.hubspot.slack.client.methods.params.conversations;
+
+import java.util.Optional;
+
+import org.immutables.value.Value.Immutable;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.hubspot.immutables.style.HubSpotStyle;
+
+@Immutable
+@HubSpotStyle
+@JsonNaming(SnakeCaseStrategy.class)
+public interface ConversationsRepliesParamsIF {
+  Optional<String> getChannel();
+  Optional<String> getTs();
```

Comment:
Reading https://api.slack.com/methods/conversations.replies, it looks like `ts` is a required field